Demo/Demo/Info.plist

```diff
@@ -2,8 +2,6 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
-	<key>CFBundleShortVersionString</key>
-	<string>1.0.0</string>
 	<key>UIApplicationSceneManifest</key>
```

Comment:
Why is this deleted? I don't think it matters for TestFlight but curious

Reply:
I know Xcode can handle automatically incrementing the version # based on what exists in TestFlight, which is nice since with every upload we make we don't have to make any source code changes.

I think if Xcode deleted it, we can assume its fine